src/main/java/org/bookstore/model/Book.java

```diff
@@ -0,0 +1,38 @@
+package org.bookstore.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import java.math.BigDecimal;
+import lombok.Data;
+
+@Data
```

Comment:
dont use data along with entity annotation as it cause bugs
should be replaced with getter/setter